lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -65,9 +65,10 @@ export class ApiError extends Error {
  * @param {Object} options
  *   @param {string} options.path - The `/api/...` path of the endpoint to call
  *   @param {string} options.authToken
+ *   @param {Record<string, string>} [options.params] - Query parameters
```

Comment:
This is a trivial thing to gripe about, but the options parameter ordering is inconsistent between here and the function itself.

Reply:
Ah, indeed. Will fix.